ansible/roles/airflow/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+---
+- set_fact:
+    svc_passwd:
+      - "{{ passwd.afredis }}"
+      - "{{ passwd.afpsql }}"
+      - "{{ passwd.afsched }}"
+      - "{{ passwd.afwebsrv }}"
+      - "{{ passwd.afworker }}"
+      - "{{ passwd.plpsql }}"
+      - "{{ passwd.plshovel}}"
+- group: name={{ item.login }} gid={{ item.id }} state=present
+  with_items: "{{ svc_passwd }}"
+- user:
+    name: "{{ item.login }}"
+    group: "{{ item.login }}"
+    uid: "{{ item.id }}"
+    comment: "{{ item.comment }}"
+    shell: /usr/sbin/nologin
+    createhome: no
+    home: /nonexistent
+    state: present
+  with_items: "{{ svc_passwd }}"
+
+- name: shared tmp directory to workaround lack of tmpfs support in docker_container
+  file: dest=/srv/tmp state=directory owner=root group=root mode=0750
+
+- name: config file directory
+  file: dest=/srv/etc state=directory owner=root group=root mode=0750
+
+- name: docker network for airflow
+  docker_network:
+    name: af
+    driver_options:
+      com.docker.network.bridge.name: braf
+    ipam_options:
+      subnet: 172.26.43.0/24
+      gateway: 172.26.43.1
+
+- name: redis .../etc directory
+  file: dest=/srv/etc/af-redis state=directory owner=root group=root mode=0755
+- name: redis .../etc/redis.conf
+  template: dest=/srv/etc/af-redis/redis.conf src=af-redis.conf mode=0640 owner=root group=afredis
+- name: redis /data directory
+  file: dest=/srv/af-redis state=directory owner=afredis group=afredis mode=0750
+- name: docker run redis
+  docker_container:
+    image: redis:3.2
+    name: af-redis
+    hostname: af-redis
+    networks: [{name: af}]
+    purge_networks: true # drop `bridge` network
+    volumes:
+      - /srv/etc/af-redis:/usr/local/etc:ro
+      - /srv/af-redis:/data:rw
+    command: /usr/local/etc/redis.conf
+    user: "{{ passwd.afredis.id}}:{{ passwd.afredis.id}}"
+    restart_policy: on-failure
```

Comment:
This should be restart_policy: always if we want that the docker container restarts on boot